crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -371,8 +371,17 @@ impl ScheduleGraph {
     }
 
     fn add_systems<P>(&mut self, systems: impl IntoSystemConfigs<P>) {
-        let SystemConfigs { systems, chained } = systems.into_configs();
-        let mut system_iter = systems.into_iter();
+        let SystemConfigs {
```

Comment:
what's the difference between `add_systems` and `configure_set` with this change?

Reply:
I am not exactly sure what you mean, because I would say the difference between these two did not change.
`add_systems` operates on an implicit system set defined by the added systems and `configure_set` on an explicitly passed system set, but without adding any systems.

Reply:
if run_systems is just using configure_set under the hood now is there possibly a way to combine the apis?

Reply:
Hm, I don't think so. You can't combine this approach with a system set that is accessible/passed from outside, because it would have unexpected side effects. E.g. the current  `add_systems((a,b).after(c).in_set(d))` would not be equal to an adjusted `add_systems((a, b).after(c), d)` when we use `d` instead of the current anonymous set, because `d` after `c` is only implied by the latter. But maybe I am still misunderstanding something.